services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/BasePreservedKeyChecker.java

```diff
@@ -16,6 +21,16 @@ public boolean isKeyPreserved(String key) {
     return IS_OH_PREFIXED.test(key);
   }
 
+  /**
+   * Details of determining if a key is preserved for the given {@link TableDto} is hidden in the
+   * implementation of aspect that associates with the annotation {@link TblPropsEnabler}
```

Comment:
nit: `.. implementation of aspect {@link FeatureToggleAspect}`

Reply:
addressed.